filters/all.js

```diff
@@ -531,6 +539,69 @@ function getAdditionalSubs(asyncapi, params) {
   return ret;
 }
 
+function getBrokerSettings(asyncapi, params){
+ let brokerSettings;
+
+ if(params.useServers === 'true'){
+    brokers = "";
+
+
+    for ( serverName in asyncapi.servers() ){
+       server = asyncapi.servers()[serverName];
+       let url = server.url()
+       if (server.variables()) {
+         url = replaceVariablesWithValues(url, server.variables());
+       }
+       brokers += `${url},`;
+     }
+     brokers = brokers.substring(0, brokers.length - 1);
+     brokerSettings = {};
+     brokerSettings.binder = {};
+     brokerSettings.binder.brokers = brokers;
+ }
+ return brokerSettings;
+}
+
+
+function replaceVariablesWithValues(url, serverVariables) {
```

Comment:
Will remove this function once this PR is solved: https://github.com/asyncapi/generator-filters/pull/46

Reply:
There is also a code smell in this function, which will be fixed once this function is removed.